homeassistant/components/cloud/prefs.py

```diff
@@ -216,20 +263,20 @@ def alexa_enabled(self):
     @property
     def alexa_report_state(self):
         """Return if Alexa report state is enabled."""
-        return self._prefs.get(PREF_ALEXA_REPORT_STATE, DEFAULT_ALEXA_REPORT_STATE)
```

Comment:
The old code would always take the default as it was today. The new code will apply the default as the chosen preference.